### PR TITLE
feat: add get did method

### PR DESCRIPTION
**Pull request by paulobmarcos**

### CODE REVIEW

**Comment by paulobmarcos** [comment]
> Missing readme update right?

Yes, went straight to it after opening the PR 👍

File: README.md [hunk 1]
```diff
@@ -70,6 +73,24 @@ const didDocument = await ipid.update(pem, (document) => {
...
 
+### getDid(pem)
+
+Returns the DID associated to a private key in PEM format.
+
+#### pem
+
+Type: `String`
+
+A private key in PEM format.
```
Comment (satazor) [inline README.md]: Should we make it clear everywhere we mention pem that we support pkcs1 and pkcs8?

Comment (paulobmarcos) [reply README.md]: A tested with every PEM format for private keys available in [crypto-keys-composer](https://github.com/ipfs-shipyard/js-crypto-key-composer/) and it accepted all of them.

Comment (paulobmarcos) [reply README.md]: Added `pkcs1-pem` and `pkcs8-pem` as the supported formats.

**Pull request merged**